### DISCUSSION TITLE
3D model looks wrong after usdz auto-conversion

### DISCUSSION THREAD

**Discussion by codedotart** [google/model-viewer#2535] [General]
Hi,

I am working with a chair model and it becomes disfigured when opened in quick look inside ios. You can see it here

https://teal-whip-brie.glitch.me

**IOS**

![WhatsApp Image 2021-07-02 at 12 00 52 AM](https://user-images.githubusercontent.com/26019854/124175983-e92a1e80-dacb-11eb-855e-63cfb7300bc8.jpeg)

**Android**

![WhatsApp Image 2021-07-02 at 12 02 14 AM](https://user-images.githubusercontent.com/26019854/124176015-f47d4a00-dacb-11eb-9d39-dfcd88808a5e.jpeg)

Can anyone please help me in resolving this issue?

**Comment by elalish** [comment]
Yep, this looks like a bug or unsupported feature in USDZExporter. @mrdoob any idea if this is a known issue or not? 

To work around issues like this until they're fixed, you can supply your own `ios-src`.

**Reply by codedotart** [reply]
If I have to provide ios-src then I will lose the material variant feature :(
Maybe I can change the mesh in a way to bypass this issue.

**Reply by mrdoob** [reply]
The problem here is that `Plastic_Base` has a negative scale.

![Screenshot 2021-07-02 at 20 15 27](https://user-images.githubusercontent.com/97088/124319934-e1718500-db72-11eb-8062-7ff0fdd745b7.png)

I'm going to add a warning in the exporter so we can easily spot these.

**Reply by mrdoob** [reply]
https://github.com/mrdoob/three.js/commit/a40bb1ae1d384c039d5311462404c93b0acef6ba

**Reply by elalish** [reply]
@mrdoob Thanks! I wonder if we're getting negative scales from any design software that uses a left-handed coordinate system? Unfortunately there are some major ones, so we may want to consider a work-around where we do the triangle inversion in the export process.

**Reply by mrdoob** [reply]
I considered doing that but the same geometry may be used by a different mesh.
So I opted for logging a warning for now.

**Reply by codedotart** [reply]
Great! thanks for the help. We fixed it now. Also, the alpha map is not working on ios ar. Is that something that is not supported yet?

**Reply by mrdoob** [reply]
https://github.com/mrdoob/three.js/pull/22086
